firmware/user/user_main.c

```diff
@@ -76,40 +76,41 @@ os_event_t procTaskQueue[PROC_TASK_QUEUE_LEN] = {{0}};
 
 swadgeMode* swadgeModes[] =
 {
+    /* This MUST NOT be defined for production firmware. It should only be
+     * enabled from the makefile when building test firmware for manufacturing.
+     * It uses all hardware features to quickly validate if hardware is functional.
+     * This comes before menuMode so the menu does not have to be navigated.
+     */
 #ifdef TEST_MODE
-#if SWADGE_VERSION != SWADGE_BBKIWI
     &testMode,
 #endif
-#endif
+    /* SWADGE_2019 doesn't have an OLED, so this is useless.
+     * For all other swadges, it comes first so the swadge boots into the menu.
+     */
 #if SWADGE_VERSION != SWADGE_2019
-    &menuMode, // Menu must be the first
-#endif
-#ifdef TEST_MODE
-#if SWADGE_VERSION == SWADGE_BBKIWI
```

Comment:
it WAS convenient to use the test mode for little tests of my own.

Reply:
Sorry, I didn't see any changes in test mode, so I didn't know you were using it. It was meant for manufacturing test, not development test, but I guess it works there too! This is also the sort of thing where it's good to leave a comment so other devs know what's going on.